### PR TITLE
Added Github Actions

### PR DESCRIPTION
**Pull request by mtekman**
Thanks to @OlegZharkov for the primer and @simonbray for the template

### CODE REVIEW

File: .github/workflows/c-cpp.yml [hunk 1]
```diff
@@ -30,3 +30,11 @@ jobs:
         make && make install
         # call binary
         $HOME/ExpaRNA/bin/ExpaRNA -h
+    - name: Test
+      run: |
+        $HOME/ExpaRNA/bin/ExpaRNA -v -e -a -i -o -L -s 2 Examples/16SrRNA_Ecoli_Ddiscoid.fa
+    - name: Check outputs
```
Comment (bgruening) [inline .github/workflows/c-cpp.yml]: shouldn't we move this into one job?

Comment (mtekman) [reply .github/workflows/c-cpp.yml]: Yep, will do

**Comment by bgruening** [comment]
Can you remove the travis file as well? Thanks.

**Comment by mtekman** [comment]
done, I squashed some commits too

**Comment by bgruening** [comment]
Lets get this in and see :) I will also remove te travis hooks then.

**Comment by bgruening** [comment]
Ok. If you like try it with some trivial change if you like.

Thanks!

**Pull request merged**